lib/linguist/languages.yml

```diff
@@ -2066,6 +2066,7 @@ HTML+ERB:
   extensions:
   - ".erb"
   - ".erb.deface"
+  - ".rhtml"
```

Comment:
I recommend adding `rhtml` to `HTML+ERB`'s aliases list:

~~~diff
 HTML+ERB:
   type: markup
   tm_scope: text.html.erb
   group: HTML
   aliases:
   - erb
+  - rhtml
~~~